packages/integration-tests/src/__tests__/integration/mysql/__snapshots__/introspection.test.ts.snap

```diff
@@ -217,7 +217,7 @@ datasource mysql {
 
 model users {
   id    BigInt @id @default(autoincrement()) @mysql.UnsignedBigInt
-  email String @unique @mysql.VarChar(50)
+  email String @unique(map: "email") @mysql.VarChar(50)
```

Comment:
Which SQL is this connected to?

Reply:
https://github.com/prisma/prisma/blob/2e70c5c20d6dac6572f3f1850381535f6af6dcba/packages/integration-tests/src/__tests__/integration/mysql/__scenarios.ts#L399-L406

Reply:
Interesting, so `email varchar(50) not null unique` leads to the unique being named `email` instead of the database default and it being rendered?

cc @do4gr - not sure if we were aware of this case yet

Reply:
I think it's expected on MySQL

Reply:
This is the Mysql default for unnamed uniques I think